boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_defconfig

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_GPIO=y
+#CONFIG_GPIO_AW9523B=y
```

Comment:
if not needed, get rid of

Reply:
> if not needed, get rid of

yeah, I think it should be better to remove them before the driver merged. I will keep them in a test branch.